app/views/shared/_footer.html+new_navigation.erb

```diff
@@ -0,0 +1 @@
+<%= render 'govuk_component/document_footer', @content_item.document_footer_for_new_navigation %>
```

Comment:
Okay, so I've had a look at this, but I think if we wind up doing this, we have to start passing flags from the controllers to the presenters, and you start scattering your A/B logic between controllers, views/partials _and_ presenters (whereas right now all A/B logic exists in just controllers and views/partials). I think this third place for logic obscures the code a bit, which outweighs the risk of things drifting, especially since this code should be merged into actual prod code (or not) over the next couple of months anyway.